builtin/sparse-checkout.c

```diff
@@ -12,6 +12,7 @@
 #include "lockfile.h"
 #include "resolve-undo.h"
 #include "unpack-trees.h"
+#include "wt-status.h"
```

Comment:
(In case anyone is paying attention, this commit is actually a _revert_ of the commit `Revert "sparse-checkout: check for dirty status"` which will get obliterated in the next rebase with this fixup.)